posthog/api/utils.py

```diff
@@ -125,11 +125,10 @@ def format_paginated_url(request: request.Request, offset: int, page_size: int,
 
 def get_token(data, request) -> Optional[str]:
     token = None
-    if request.method == "GET":
```

Comment:
Mobile SDK was passing the `api_key` as a query param but in a POST request. Unsure if this `"GET"` condition is necessary but we should consider removing it in future